src/tpt_reports/tpt_reports.py

```diff
@@ -29,13 +29,13 @@
 # Third-Party Libraries
 import docopt
 from schema import And, Schema, SchemaError, Use
+from validator_collection import validators
 
 from ._version import __version__
 from .report_generator import report_gen
 
 LOGGER = logging.getLogger(__name__)
 LOGGER.addHandler(logging.StreamHandler())
-LOGGING_FILE = "report_generator.log"
```

Comment:
Did you mean to remove the logging file in this PR?

Reply:
Yes. We removed it because it is unused right now but will be added back in for Issue: https://github.com/cisagov/tpt-reports/issues/40

Reply:
Makes sense - thanks for the explanation.